crates/bevy_sprite/src/sprite.rs

```diff
@@ -19,6 +21,33 @@ pub struct Sprite {
     pub anchor: Anchor,
 }
 
+/// Component for sprite sheets containing a handle to [`TextureAtlas`] and the index of the current
+/// section of the sheet
+#[derive(Component, Default, Debug, Clone, Reflect)]
+pub struct SpriteSheet {
+    /// Texture atlas handle
+    pub texture_atlas: Handle<TextureAtlas>,
+    /// Texture atlas section index
```

Comment:
I would like a little clearer docs here. What exactly is the index pointing to?

Reply:
A section (`Rect`) of the `TextureAtlas` I don't know how to word it better

Reply:
I like that explanation, but we should add this information to the docs ;)

Reply:
Done !